llvm/test/CodeGen/SPIRV/basic_float_types.ll

```diff
@@ -0,0 +1,49 @@
+; RUN: llc -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s
```

Comment:
BTW, maybe we could expand coverage to other SPIR-V versions by adding additional RUN lines?
Isn't adding the other triples enough to extend coverage?

```
; RUN: llc -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s
; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s
; RUN: llc -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
```

Reply:
Done